From b0ae24af94ef5ab695d0f5c3e214bd8b7baac157 Mon Sep 17 00:00:00 2001
From: Gao Pan <gaopan@freescale.com>
Date: Thu, 11 Jun 2015 16:45:47 +0800
Subject: [PATCH 0987/1691] MLK-11402 ARM: dts : add i2c IOMUX sleep state

commit b0ae24af94ef5ab695d0f5c3e214bd8b7baac157 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Add i2c IOMUX sleep state for imx7d-12x12-lpddr3-arm2.

Signed-off-by: Gao Pan <b54642@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index 59639a0..0617ee3 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -230,9 +230,10 @@
 
 &i2c1 {
 	clock-frequency = <100000>;
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_i2c1_1>;
-	status = "disabled";
+	pinctrl-1 = <&pinctrl_i2c1_1>;
+	status = "okay";
 
 	pmic: pfuze3000@08 {
 		compatible = "fsl,pfuze3000";
@@ -327,9 +328,10 @@
 
 &i2c3 {
 	clock-frequency = <100000>;
-	pinctrl-names = "default";
+	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&pinctrl_i2c3_1>;
-	status = "disabled";
+	pinctrl-1 = <&pinctrl_i2c3_1>;
+	status = "okay";
 
 	max7322: gpio@68 {
 		compatible = "maxim,max7322";
-- 
1.9.1

